### PR TITLE
Kill switch for Community care searches on Facility Locator

### LINKED ISSUE #28782: Kill switch for Community care searches on Facility Locator
## Issue Description
As a Veteran, I want to be informed if searches for community care providers are temporarily unavailable so that I know to try my search later. 

We currently have two flags, one for pharmacies and one for community care. Turning those on removes the corresponding option from the Facility Type dropdown.

## Future work (separate issues)

- Remove unneeded toggles 
- Create endpoint-specific kill switches 
- Consider automated kill switch based on breaker

---
### Design
- [x] Identify the correct style of error message to display to inform Veterans that 

> We’re sorry. All community care searches, including urgent care, emergency care, pharmacies and,  community providers, are currently unavailable. Please try your search again later. 

### Engineering
- [ ] Implement error message display - [Link to UX Pin](https://preview.uxpin.com/8c3f8f1129d12d7a991c30aa31b856fd2d940227#/pages/141142822/simulate/sitemap)
- [ ] Create a single kill switch which disables all community care searches, including 
       - Facility type = Community providers
       - Facility type = urgent care 
       - Facility type = emergency care
       - Facility type = community pharmacies
- [ ] Add flipper to allow this functionality to be quickly toggled in Production
- [ ] Add error message with [language tbd]

## Acceptance criteria
- [ ] Facilities team is able to disable all community care searches with a single toggle
- [ ] Veterans are notified that the searches are temporarily available via error message with style specified below
- [ ] Community care options are not available in the facility type dropdown

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/28782

### CODE REVIEW

**Review by zacharymorel — APPROVED** [review]

File: src/applications/facility-locator/components/Alert.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Copied and tweaked from src/applications/claims-status/components/appeals-v2/Alert.jsx.
+ * This should probably become a shared component.
```
Comment (oseasmoran73) [inline src/applications/facility-locator/components/Alert.jsx:6]: Might make sense to add a `TODO` so it won't slip under the radar

**Review by oseasmoran73 — APPROVED** [review]
LGTM. Please verify lint warnings and if actions are needed, be done so accordingly

**Review by gescobardev — APPROVED** [review]

File: src/applications/facility-locator/components/Alert.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    cssClass = 'usa-alert-info';
+  }
+
+  return (
+    <div
+      className={`usa-alert ${cssClass} vads-u-margin-top--0 vads-u-margin-bottom--2 ${
+        isMobile
+          ? 'vads-u-margin-right--3 vads-u-margin-left--2 vads-u-width--auto'
+          : ''
+      }`}
```
Comment (kelsonic) [inline src/applications/facility-locator/components/Alert.jsx:28]: Hey MPS! I think you can just do this if you want 🙂 

```
className="usa-alert vads-u-margin-right--3 vads-u-margin-left--2 vads-u-width--auto medium-screen:vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--2"
```

Comment (kelsonic) [inline src/applications/facility-locator/components/Alert.jsx:28]: Hey MPS! I think you can just do this if you want 🙂 

```
className="usa-alert ${cssClass} vads-u-margin-right--3 vads-u-margin-left--2 vads-u-width--auto medium-screen:vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--2"
```

**Pull request merged**